apiserver/tools_test.go

```diff
@@ -165,12 +165,12 @@ func (s *toolsSuite) TestUploadFailsWithInvalidContentType(c *gc.C) {
 		c, resp, http.StatusBadRequest, "expected Content-Type: application/x-tar-gz, got: application/octet-stream")
 }
 
-func (s *toolsSuite) setupToolsForUpload(c *gc.C) (coretools.List, version.Binary, string) {
+func (s *toolsSuite) setupToolsForUpload(c *gc.C) (coretools.List, string, string) {
```

Comment:
I think it's actually perhaps slightly nicer with version.Binary as that shows what we're expecting it to be.